src/components/learner-credit-management/AssignmentStatusTableCell.jsx

```diff
@@ -16,12 +17,16 @@ const AssignmentStatusTableCell = ({ row }) => {
     learnerState,
     errorReason,
   } = original;
-
   // Learner state is not available for this assignment, so don't display anything.
   if (!learnerState) {
     return null;
   }
 
+  // If learnerState is failed but no error reason is defined, return a failed system chip.
+  if (learnerState === 'failed' && !errorReason) {
+    return <FailedSystem />;
```

Comment:
Note, there is a `learnerState === 'failed'` condition below where this could go (see line 43).

Reply:
+1